ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,179 @@
+1171.6 "Accursed Visage" sync / 1[56]:[^:]*:Gladiator of Sil'dih:768D:/
+1176.7 "Wrath of Ruin" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7663:/
+1185.7 "Gold Flame and Silver Flame" sync / 1[56]:[^:]*:Hateful Visage:(766F|7670):/
+1187.3 "Nothing beside Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:(768B|768C):/
+1196.4 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1206.6 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7639:/
+1212.7 "Curse of the Monument" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7666:/
+1219.9 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1220.5 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1221.1 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1221.7 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1222.3 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1224.9 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7639:/
+1232.9 "Scream of the Fallen" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7678:/
+1233.3 "Colossal Wreck" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7669:/
+1233.8 "Explosion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766A:/
+1237.0 "Scream of the Fallen" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7678:/
+1237.8 "Explosion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766A:/
+1245.0 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1259.2 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1273.9 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1276.0 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1278.3 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1293.0 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1295.1 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1297.5 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:6854:/
+1260.0 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766C:/
+1260.3 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:79F3:/
+
+#~~~~~~~~~~~~~~~~~~~~~~~~~#
+# Shadowcaster Zeless Gah #
+#~~~~~~~~~~~~~~~~~~~~~~~~~#
+
+2000.0 "--sync--" sync / 00:0839::The Trial of Balance will be sealed off/ window 2000,1
+2006.8 "--sync--" sync / 14:[^:]*:Shadowcaster Zeless Gah:74AF:/ window 1007.3,5
+2011.5 "Show of Strength" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74AF:/
+2023.7 "Infern Brand" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:7491:/
+2030.9 "Cryptic Portal" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:7494:/
+2042.0 "Firesteel Strike" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B0:/
+2042.4 "Firesteel Strike" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B1:/
+2043.8 "Firesteel Strike" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B2:/
+2051.8 "Blessed Beacon" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B3:/
+2052.2 "Blessed Beacon" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B4:/
+2054.3 "Blessed Beacon" sync / 1[56]:[^:]*:Shadowcaster Zeless Gah:74B5:/
```

Comment:
Should these be shortened or some removed?

Reply:
Yea, looks like there is some paste errors here.